launcher/Application.cpp

```diff
@@ -986,6 +988,16 @@ void Application::performMainStartupAction()
             return;
         }
     }
+    if(!m_instanceIdToShowWindowOf.isEmpty())
+    {
+        auto inst = instances()->getInstanceById(m_instanceIdToShowWindowOf);
+        if(inst)
```

Comment:
This doesn't have a behavior defined for when an invalid ID is used, so it just continues as if launching prism as normal. Maybe a more intuitive behavior would be to refuse to launch the program, like when `--show` is used without arguments, specifying something like "Invalid ID given to --show"

Reply:
I'm aware, but I modelled how `--show` works based on how `--launch` works (defined just a few lines above), it works perfectly fine with a valid ID, drops you onto the main window otherwise with no error or warning. Should I not have done so?

Reply:
ehh, i think this behavior with `--launch` is also weird :p

well, since it's already like that, no need to change it.